.github/workflows/fuzz.yml

```
@@ -0,0 +1,32 @@
+name: Fuzz Workflow
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)